posts/dialog.md

```diff
@@ -0,0 +1,12 @@
+feature: dialog
```

Comment:
I'd prefer to be [consistent with other tags here](https://raw.githubusercontent.com/h5bp/html5please/master/posts/details.md), and wrap the element name in html-like syntax.